### ISSUE TITLE
Explore using Cypress for Accessibility Testing

### ISSUE THREAD

**Issue by ricardocarva** [ricardocarva/MortiSmart#43]
Raising this issue to explore implementing Cypress Accessibility Testing in our CircleCI pipeline to check for Accessibility violations.

The following link includes some details on the tool and how to set it up: https://circleci.com/blog/cypress-accessibility-testing/
